04-kogito-travel-agency/README.md

```diff
@@ -158,6 +158,28 @@ To run the generated native executable, generated in `target/`, execute
 ./target/kogito-travel-agency-{version}-runner
 ```
 
+### Start Kogito Data Index Service
+
+ We have to start the Data Index Service, we can find this service description more detailed at [Kogito Data Index Service](https://github.com/kiegroup/kogito-runtimes/wiki/Data-Index-Service).
+  
+ It can be downloaded from [Kogito Dada Index Service 0.4.0](http://repo2.maven.org/maven2/org/kie/kogito/data-index-service/0.4.0/data-index-service-0.4.0-runner.jar)
```

Comment:
maybe just add additional note so people can find other versions as well 

````
NOTE: Here can be found other versions [Kogito Dada Index Service versions](http://repo2.maven.org/maven2/org/kie/kogito/data-index-service)
````

Reply:
sure, it makes sense, added!